src/service/brokers/http/connection.toit

```diff
@@ -18,8 +18,9 @@ class HttpConnection_:
 
   constructor .network_ .config_:
     if config_.root-certificate-ders:
-      root-certificates_ = config_.root-certificate-ders.map:
-        x509.Certificate.parse it
+      config_.root-certificate-ders.do:
+        certificate := tls.RootCertificate it
+        certificate.install
```

Comment:
Can we do this early when setting up the config instead? There is no hash for this one (I think), so we're going to parse it every single time.

Reply:
It's hard to do this earlier, as the Broker is constructed relatively early, but we don't want to spend time installing roots if they won't be needed.
I added a flag that keeps track of whether they are already installed or not.